meta-mender-core/recipes-mender/mender-client/mender-client_2.5.0.bb

```diff
@@ -1,4 +1,4 @@
-require mender-client.inc
+require mender-client-pre-3.1.inc
```

Comment:
It should be pre 3.2, right? Mender client 3.1 we are tentatively releasing in few weeks from now, and this feature goes into the following one, 3.2.

Reply:
Yes! Good catch!